spec/features/post_an_ad_spec.rb

```diff
@@ -17,6 +17,10 @@ def verify_ad_is_visible
     expect(page).to have_content("044555555")
   end
 
+  before do
+    create(:city, name: "Petrinja")
+  end
```

Comment:
You needed to do this because your db was empty, right? CI runs it after running `RAILS_ENV=test bin/rails db:setup` so it's not empty.

Reply:
I've changed that https://github.com/vlado/earthquake-croatia/pull/131/files#diff-b32b798d8ec577b859922a84b90dfe871b19fad404cc6fa7a6fe163f079fbcbfR103 so we do not load seed data in test database.

I think it is surprising (at least it was for me) and didn't realised right away why are my specs failing on CI. Also it requires developers to load seed data to test env locally.

Reply:
I agree with the change to factories. 👍